src/components/interactive/Avatar.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import baseElement from '../../util/baseElement';
+
+export const VALID_BG_COLORS = [
+  'white',
+  'haze',
+  'lightGray',
+  'orange',
+  'charcoal',
+  'navy',
+  'aqua',
+  'gray',
+  'red',
+];
+
+const INVERTED_BG_COLORS = ['gray', 'charcoal', 'navy', 'orange'];
```

Comment:
Instead of whitelisting colors to invert, I basically changed this to blacklist things that shouldn't be inverted:

`const NON_INVERTED_BG_COLORS = ['white', 'haze', 'lightGray'];`